metricbeat/module/ceph/_meta/config.yml

```diff
@@ -1,18 +1,18 @@
 - module: ceph
-  #metricsets:
-  #  - cluster_health
-  #  - cluster_status
-  #  - monitor_health
+  metricsets:
+    - cluster_health
+    - cluster_status
+    - monitor_health
   period: 10s
   hosts: ["localhost:5000"]
   #username: "user"
   #password: "secret"
 
 - module: ceph
-  #metricsets:
-  #  - cluster_disk
-  #  - osd_tree
-  #  - osd_df
-  #  - pool_disk
+  metricsets:
```

Comment:
I think this was actually a "bug" before as it meant the module is started twice with all default metricsets. Should we backport this fix to 6.3?